mpt/main.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+import datetime
+
+import crawler
+
+try:
+    month = os.environ['MONTH']
+    year = os.environ['YEAR']
+    path_driver = os.environ['DRIVER']
+except KeyError:
+    sys.stderr.write("Invalid arguments, missing parameters.\n")
```

Comment:
Poderia ser mais específico e dizer qual variável está vazia?

Lembrando que você pode usar `if 'MONTH' in os.environ`